### PR TITLE
new translation work on 1101

### PR DESCRIPTION
**Pull request by HENGRui6**
[oneflow.Tensor.cup](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/tensor_ops.py#L189)
38dad788feb71bea72893b4b4d28170

[oneflow.Tensor.cuda](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/tensor_ops.py#L208) 
91ea1ae3ea61ec24904589397a14a35

[oneflow.Tensor.double](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/tensor_ops.py#L144) 
1bab040af96f6d52f527c2b8f990419

[oneflow.Tensor.expand_as](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/framework/docstr/tensor.py#L64) 
d50461a061c0f754bc8cab831a4a120

[oneflow.Tensor.float](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/tensor_ops.py#L122) 
7382a524a5fd9a380e1449f13c5817e

[oneflow.gather](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/gather.py#L22) 
545618c0ec2f6174c00952a311f4d73

[oneflow.Tensor.int](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/tensor_ops.py#L78) 
e0f3b2bd33524f79b0d18a6707f4956

[oneflow.Tensor.item](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/tensor_ops.py#L233) 
0b38ba05c14cfc11aac7a32b3979f57

[class oneflow.nn.AdaptiveAvgPool1d](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/pooling.py#L591) 
9ca36b7b764cc2a198ca7b42c731df4

[class oneflow.nn.AdaptiveAvgPool2d](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/pooling.py#L643) 
e4fcdede5002d4cb460246f02ba1d68

[class oneflow.nn.AdaptiveAvgPool3d](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/pooling.py#L708) 
f8df0d040cd403c7011647399f4e5d7

[class oneflow.nn.AvgPool1d](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/pooling.py#L348) 
e744466154d4601b2fd2c88e103b940

[class oneflow.nn.AvgPool2d](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/pooling.py#L425) 
6d01200842773f4e1a785d4d1712dcc

[class oneflow.nn.AvgPool3d](https://github.com/Oneflow-Inc/oneflow/blob/2b8011598fc142364ca9fd92b04dbbd5dcfad8cf/python/oneflow/nn/modules/pooling.py#L498) 
a946bba19ea0b700bed343b70843436

[oneflow.Tensor.reciprocal](https://github.com/Oneflow-Inc/oneflow/blob/82f58ee971c2d8c55722c1a49baa49c6cfb0b05f/python/oneflow/nn/modules/math_ops.py#L115) 
933b4a0d48fa5a7560555bd8bf4e222

[oneflow.linalg.norm](https://github.com/Oneflow-Inc/oneflow/blob/82f58ee971c2d8c55722c1a49baa49c6cfb0b05f/python/oneflow/framework/docstr/norm.py#L161)
d680a635cbf434f5f2a1426b61f9d4d
a59c2e412cbe2d4460bb75309b07ce6

[oneflow.Tensor.new_ones](https://github.com/Oneflow-Inc/oneflow/blob/82f58ee971c2d8c55722c1a49baa49c6cfb0b05f/python/oneflow/nn/modules/constant.py#L338)
9aa334d7ddf62319082a90d70a7a1f6

[oneflow.Tensor.ne](https://github.com/Oneflow-Inc/oneflow/blob/82f58ee971c2d8c55722c1a49baa49c6cfb0b05f/python/oneflow/nn/modules/comparison.py#L145)
40e1bd4e647948ed4338405c9a0ffd9

[oneflow.Tensor.reshape](https://github.com/Oneflow-Inc/oneflow/blob/82f58ee971c2d8c55722c1a49baa49c6cfb0b05f/python/oneflow/nn/modules/reshape.py#L63)
6c2d70cf27e4b8a7d739e0c4e8366c2

### CODE REVIEW

File: docs/source/cn/pooling.py [hunk 1]
```diff
@@ -44,3 +44,112 @@
...
+        oneflow.Size([1, 64, 5])
+
+    """
+)
+
+reset_docstr(
+    oneflow.nn.AdaptiveAvgPool2d,
+    r"""AdaptiveAvgPool2d(output_size)
+    
+    在由多个平面组成的的输入信号上应用 2D 自适应平均池化。
```
Comment (BONNNIE) [inline docs/source/cn/pooling.py:80]: 其他地方对于“input signal”的翻译，似乎都是“信号 `input`”，感觉统一一下比较好～

Comment (HENGRui6) [reply docs/source/cn/pooling.py]: 嗯嗯，已修改

File: docs/source/cn/tensor_ops.py [hunk 2]
```diff
@@ -75,3 +75,193 @@
...
+        >>> input.dtype
+        oneflow.int32
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.item,
+    r"""
+    将 tensor 的值作为标准 Python 数字返回。仅适用于只有一个元素的 tensor 。
+    其他情况请参考 :mod:`oneflow.tolist` 。
```
Comment (Kitten97) [inline docs/source/cn/tensor_ops.py:226]:
```suggestion
    将 tensor 的值作为标准 Python 数字返回。仅适用于只有一个元素的 tensor。
    
    其他情况请参考 :mod:`oneflow.tolist` 。
```

File: docs/source/cn/pooling.py [hunk 3]
```diff
@@ -44,3 +44,244 @@
...
 )
+
+reset_docstr(
+    oneflow.nn.AdaptiveAvgPool1d,
+    r"""AdaptiveAvgPool1d(output_size)
+
+    在由多个平面组成的输入信号 `input` 上应用 1D 自适应平均池化。
+
+    对于任何大小的输入，输出大小都是 H 。
+    输出的数量等于输入平面的数量。
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:55]:
```suggestion
    对于任何大小的输入，输出大小都是 H。
    
    输出的数量等于输入平面的数量。
```

File: docs/source/cn/pooling.py [hunk 4]
```diff
@@ -44,3 +44,244 @@
...
+
+    参数：
+        - **output_size** (Int64List[1]): 目标输出大小（单个整数）
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:58]: 原英文是` output_size: the target output size H` 中文要不要加上这个 H
```suggestion
        - **output_size** (Int64List[1]): 目标输出大小 H（单个整数）
```

File: docs/source/cn/pooling.py [hunk 5]
```diff
@@ -44,3 +44,244 @@
...
+)
+
+reset_docstr(
+    oneflow.nn.AdaptiveAvgPool2d,
+    r"""AdaptiveAvgPool2d(output_size)
+    
+    在由多个平面组成的的信号 `input` 上应用 2D 自适应平均池化。
+
+    对于任何大小的输入，输出大小都是 H x W 。
+    输出的数量等于输入平面的数量。
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:83]:
```suggestion
    在由多个平面组成的的信号 `input` 上应用 2D 自适应平均池化。

    对于任何大小的输入，输出大小都是 H x W。
    
    输出的数量等于输入平面的数量。
```
如果中间不空行，渲染的效果就多了个空格，很别扭，所以我们就都加上空行吧~

File: docs/source/cn/pooling.py [hunk 6]
```diff
@@ -44,3 +44,244 @@
...
+    在由多个平面组成的信号 `input` 上执行 1D 平均池化。
+    在最简单的情况下，输出值是输入大小为 :math:`(N, C, H, W)` 的层。
+    输出 :math:`(N, C, H_{out}, W_{out})` 和 `kernel_size` ， :math:`k` 可以被精确地描述为：
+    
+    .. math::
+        out(N_i, C_j, l)  = \frac{1}{k} \sum_{m=0}^{k-1}
+                            input(N_i, C_j, stride[0] \times h + m, stride*l + m)
+    
+    如果 :attr:`padding` 非零，则输入在两侧隐式填充 0 以填充点数。
+    参数 :attr:`kernel_size` 、 :attr:`stride` 、 :attr:`padding` 可以为 int 或者单元素元组。
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:170]:
```suggestion
    如果 :attr:`padding` 非零，则输入在两侧隐式填充 0 以填充点数。
    
    参数 :attr:`kernel_size` 、 :attr:`stride` 、 :attr:`padding` 可以为 int 或者单元素元组。
```

File: docs/source/cn/pooling.py [hunk 7]
```diff
@@ -44,3 +44,244 @@
...
+    
+    Note:
+        当 :attr:`ceil_mode` 为 True 时，如果滑动窗口在 left padding 或输入内开始，则允许滑动窗口出界。
+        忽略在右侧填充区域开始的滑动窗口。
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:174]:
```suggestion
        当 :attr:`ceil_mode` 为 True 时，如果滑动窗口在 left padding 或输入内开始，则允许滑动窗口出界。忽略在右侧填充区域开始的滑动窗口。
```

File: docs/source/cn/pooling.py [hunk 8]
```diff
@@ -44,3 +44,244 @@
...
+)
+
+reset_docstr(
+    oneflow.nn.AvgPool2d,
+    r"""AvgPool2d(kernel_size, stride=None, padding=0, ceil_mode=False, count_include_pad=True, divisor_override=0)
+    
+    在由多个平面组成的信号 `input` 上执行 2D 平均池化。
+
+    在最简单的情况下，输出值是输入大小为 :math:`(N, C, H, W)` 的层。
+    输出 :math:`(N, C, H_{out}, W_{out})` 和 `kernel_size` ， :math:`(kH, kW)` 可以被精确地描述为：
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:205]:
```suggestion
    在最简单的情况下，输出值是输入大小为 :math:`(N, C, H, W)` 的层。
    
    输出 :math:`(N, C, H_{out}, W_{out})` 和 `kernel_size` ， :math:`(kH, kW)` 可以被精确地描述为：
```

File: docs/source/cn/pooling.py [hunk 9]
```diff
@@ -44,3 +44,244 @@
...
+    """
+)
+
+reset_docstr(
+    oneflow.nn.AvgPool3d,
+    r"""AvgPool3d(kernel_size, stride=None, padding=0, ceil_mode=False, count_include_pad=True, divisor_override=0)
+
+    在由多个平面组成的信号 `input` 上执行 3D 平均池化。
+    在最简单的情况下，输出值是输入大小为 :math:`(N, C, D, H, W)` 的层。
+    输出 :math:`(N, C, D_{out}, H_{out}, W_{out})` 和 `kernel_size` ， :math:`(kD, kH, kW)` 可以被精确地描述为：
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:239]:
```suggestion
    在由多个平面组成的信号 `input` 上执行 3D 平均池化。在最简单的情况下，输出值是输入大小为 :math:`(N, C, D, H, W)` 的层。
    
    输出 :math:`(N, C, D_{out}, H_{out}, W_{out})` 和 `kernel_size` ， :math:`(kD, kH, kW)` 可以被精确地描述为：
```

File: docs/source/cn/pooling.py [hunk 10]
```diff
@@ -44,3 +44,244 @@
...
+    .. math::
+        out(N_i, C_j, d, h, w)  = \frac{1}{kD * kH * kW } \sum_{k=0}^{kD-1} \sum_{m=0}^{kH-1} \sum_{n=0}^{kW-1}
+                               input(N_i, C_j, stride[0] \times d + k, stride[1] \times h + m, stride[2] \times w + n)
+    
+    如果 :attr:`padding` 非零，则输入在三侧隐式填充 0 以填充点数。
+    
+    Note:
+
+        当 :attr:`ceil_mode` 为 True 时，如果滑动窗口在 left padding 或输入内开始，则允许滑动窗口出界。
+        忽略在右侧填充区域开始的滑动窗口。
```
Comment (Kitten97) [inline docs/source/cn/pooling.py:250]:
```suggestion
        当 :attr:`ceil_mode` 为 True 时，如果滑动窗口在 left padding 或输入内开始，则允许滑动窗口出界。忽略在右侧填充区域开始的滑动窗口。
```

File: docs/source/cn/tensor_ops.py [hunk 11]
```diff
@@ -75,3 +75,193 @@
...
+        >>> x = flow.tensor([1.0])
+        >>> x.item()
+        1.0
+    """
+)
+
+reset_docstr(
+    oneflow.Tensor.reciprocal,
+    r"""reciprocal(x) -> Tensor
+    计算 :attr:`x` 的倒数，如果 :attr:`x` 为0，倒数将被设置为0。
```
Comment (Kitten97) [inline docs/source/cn/tensor_ops.py:247]:
```suggestion
    计算 :attr:`x` 的倒数，如果 :attr:`x` 为0，倒数将被设置为 0。
```

File: docs/source/cn/tensor_ops.py [hunk 12]
```diff
@@ -75,3 +75,193 @@
...
+
+    参数：
+        **x** (Tensor): 输入张量。
```
Comment (Kitten97) [inline docs/source/cn/tensor_ops.py:250]:
```suggestion
        **x** (Tensor): 输入张量
```

File: docs/source/cn/constant.py [hunk 13]
```diff
@@ -113,6 +113,136 @@
...
+)
+
+reset_docstr(
+    oneflow.linalg.norm,
+    r"""linalg.norm(input, ord=None, dim=None, keepdim=False, *, dtype=None, out=None) -> Tensor
+
+    返回 :attr:`input` 的矩阵范数或向量范数。
+
+    此函数可以计算八种不同类型的矩阵范数之一，或无限数量的向量范数之一，
+    具体取决于减少维度的数量和 :attr:`ord` 参数的值。
```
Comment (Kitten97) [inline docs/source/cn/constant.py:151]:
```suggestion
    此函数可以计算八种不同类型的矩阵范数之一，或无限数量的向量范数之一。具体取决于减少维度的数量和 :attr:`ord` 参数的值。
```

File: docs/source/cn/constant.py [hunk 14]
```diff
@@ -113,6 +113,136 @@
...
+
+    参数：
+        - **input** (Tensor): 输入张量。如果 :attr:`dim` 是 None，则输入必须是一维或二维的，除非 :attr:`ord` 也是 None。
+            如果 :attr:`dim` 和 :attr:`ord` 都是 None ，则将返回扁平化为 1-D 的输入二范数。它的数据类型必须是浮点数或复数类型。
+            对于复杂输入，范数是根据每个元素的绝对值计算的。如果输入是复数并且既没有指定 :attr:`dtype` 也没有指定 :attr:`out`，
+            返回的数据类型将是相应的浮点类型（例如如果 :attr:`input` 是 complexfloat ，则为float ）
```
Comment (Kitten97) [inline docs/source/cn/constant.py:157]:
```suggestion
        - **input** (Tensor): 输入张量。如果 :attr:`dim` 是 None，则输入必须是一维或二维的，除非 :attr:`ord` 也是 None。
        
            如果 :attr:`dim` 和 :attr:`ord` 都是 None ，则将返回扁平化为 1-D 的输入二范数。它的数据类型必须是浮点数或复数类型。
            
            对于复杂输入，范数是根据每个元素的绝对值计算的。如果输入是复数并且既没有指定 :attr:`dtype` 也没有指定 :attr:`out`，返回的数据类型将是相应的浮点类型（例如如果 :attr:`input` 是 complexfloat ，则为float）。
```

File: docs/source/cn/constant.py [hunk 15]
```diff
@@ -113,6 +113,136 @@
...
+            `2`              -- 尚不支持 --                 as below
+            `-2`             -- 尚不支持 --                 as below
+            其他             -- 不支持 --                  `sum(abs(x)^{ord})^{(1 / ord)}`
+            ==============  ============================  =================================
+
+            其中 `inf` 指的是 `float('inf')` 、 NumPy 的 `inf` 对象或任何等效对象。
+
+        - **dim** (int, 2-tuple of ints, 2-list of ints, 可选): 如果 :attr:`dim` 是一个 int ，向量范数将在指定的维度上计算。
+            如果 :attr:`dim` 是一个整数的二元组，矩阵范数将在指定的维度上计算。如果 :attr:`dim` 为 None ，
+            则输入张量是二维张量时计算矩阵范数，输入张量为一维时计算向量范数。默认值：``None``
```
Comment (Kitten97) [inline docs/source/cn/constant.py:182]:
```suggestion
        - **dim** (int, 2-tuple of ints, 2-list of ints, 可选): 如果 :attr:`dim` 是一个 int，向量范数将在指定的维度上计算。
        
            如果 :attr:`dim` 是一个整数的二元组，矩阵范数将在指定的维度上计算。
            
            如果 :attr:`dim` 为 None，则输入张量是二维张量时计算矩阵范数，输入张量为一维时计算向量范数。默认值：``None``
```

File: docs/source/cn/constant.py [hunk 16]
```diff
@@ -113,6 +113,136 @@
...
+ 
+    放回一个 :attr:`size` 大小的张量，其元素全部为 1 。默认情况下，返回张量的 `dtype` 和 `device` 和输入张量的相同。
+
+    参数：
+        - **size** (一个整数或包含整数的元组)): 决定输出张量的形状，可以是数字变量或集合例如列表或元组
+        - **dtype** (flow.dtype, 可选): 返回张量的数据类型。
+        - **device** (flow.device, 可选): 返回的本地张量的所需设备。默认使用当前设备
+        - **placement** (flow.placement, 可选): 设置返回张量的 placement 属性。如果为None，则构造 local tensor 。
+        - **sbp** (flow.sbp.sbp 或包含 flow.sbp.sbp 的元组, 可选): 返回的consistent tensor的所需 sbp 描述符。如果为 None ，则返回的张量是使用参数 `device` 的本地张量。
+        - **requires_grad** (bool, 可选): 用 autograd 记录对返回张量的操作，默认为 False。
```
Comment (Kitten97) [inline docs/source/cn/constant.py:128]:
```suggestion
        - **size** （一个整数或包含整数的元组）：决定输出张量的形状，可以是数字变量或集合例如列表或元组
        - **dtype** （flow.dtype, 可选）：返回张量的数据类型
        - **device** （flow.device，可选）：返回的本地张量的所需设备。默认使用当前设备
        - **placement**（flow.placement, 可选）: 设置返回张量的 placement 属性。如果为 None，则构造 local tensor
        - **sbp** （flow.sbp.sbp 或包含 flow.sbp.sbp 的元组, 可选）：返回的 consistent tensor 的所需 sbp 描述符。如果为 None，则返回的张量是使用参数 `device` 的本地张量
        - **requires_grad** （bool，可选）：用 autograd 记录对返回张量的操作，默认为 False
```

File: docs/source/cn/comparison.py [hunk 17]
```diff
@@ -118,3 +118,33 @@
     
     """
 )
+
+reset_docstr(
+    oneflow.Tensor.ne,
+    r"""ne(inout, other) -> Tensor
+
+    计算 element-wise 元素不相等性。
+    第二个参数 :attr:`other` 可以是一个数字或张量，其形状可以用第一个参数 :attr:`input` 广播。
```
Comment (Kitten97) [inline docs/source/cn/comparison.py:127]:
```suggestion
    计算 element-wise 元素不相等性。
    
    第二个参数 :attr:`other` 可以是一个数字或张量，其形状可以用第一个参数 :attr:`input` 广播。
```

**Review by Kitten97 — COMMENTED** [review]
rest LGTM. 
- 括号里既有中文又有英文（即只要括号内包含任何中文）时建议使用全角括号。
- 英文或数字后为中文标点的话，就不用空格了（这个太多了，没改过来，Hengrui 自己再看看吧~）。

**Comment by HENGRui6** [comment]
> 

我觉得这种表示参数数据类型的括号不管中文英文应该统一格式吧，在这个cn文档里面参数的数据类型既有中文也有英文，如果两种的括号不统一的话看起来很乱，所以我就都用英文的括号了

File: docs/source/cn/constant.py [hunk 18]
```diff
@@ -113,6 +113,138 @@
...
 )
 
+reset_docstr(
+    oneflow.Tensor.new_ones,
+    r"""new_ones(size=None, dtype=None, device=None, placement=None, sbp=None, requires_grad=False) -> Tensor
+ 
+    放回一个 :attr:`size` 大小的张量，其元素全部为 1 。默认情况下，返回张量的 `dtype` 和 `device` 和输入张量的相同。
+
+    参数：
+        - **size** (一个整数或包含整数的元组)): 决定输出张量的形状，可以是数字变量或集合例如列表或元组
```
Comment (Kitten97) [inline docs/source/cn/constant.py:123]:
```suggestion
        - **size** (一个整数或包含整数的元组): 决定输出张量的形状，可以是数字变量或集合例如列表或元组
```

**Review by Kitten97 — APPROVED** [review]

**Comment by Kitten97** [comment]
Some issues found. Please do not merge this PR for now.

**Pull request merged**